src/Uno.UI/UI/Xaml/Internal/InputManager.cs

```diff
@@ -58,7 +58,14 @@ internal bool ShouldRequestFocusSound()
 
 	internal void NotifyFocusChanged(DependencyObject? focusedElement, bool bringIntoView, bool animateIfBringIntoView)
```

Comment:
Looks like the original nullable annotation of focusedElement may be wrong. I'd fix the annotaiton and remove the null suppression.